canonical-web-and-design/vanilla-framework
# Search results page

Updates layout of search results page.
Removes filter functionality from existing search box and replaces it with full docs search.

Fixes #2805 

## QA

See https://github.com/canonical-web-and-design/vanilla-framework/pull/2812 for instructions on how to get search API token for local use.

Note: search box will be moved to main nav separately in future PR.

- Pull code
- Run `./run`
- Open http://0.0.0.0:8101/
- Type anything to search box
- Search page with results should appear

## Screenshots

<img width="1396" alt="Screenshot 2020-02-10 at 13 20 20" src="https://user-images.githubusercontent.com/83575/74149476-1d1b5000-4c08-11ea-9104-6e7d31f120a3.png">
